src/Microsoft.AspNetCore.Server.IISIntegration/Server/IISHttpContext.cs

```diff
@@ -195,6 +196,56 @@ public string ReasonPhrase
             }
         }
 
+        public CancellationToken RequestAborted
```

Comment:
Copy from HttpSys instead of Kestrel. This is only half of the feature.

Reply:
I am hesitant to directly copy HttpSys for the disconnect listener as IIS has an OnClientDisconnect event. I'd take it you would like me to have both the abort and client disconnect changes in this PR?

Reply:
I'm saying they're separate features. You can remove RequestAborted from this PR as it's not directly related to Abort(), it's only triggered indirectly via OnClientDisconnected.